File: lib/antlr-2.7.5/examples/java/inherit.tinyc/Main.java
import java.io.*;

class Main {
	public static void main(String[] args) {
		try {
			TinyCLexer lexer = new TinyCLexer(new DataInputStream(System.in));
			MyCParser parser = new MyCParser(lexer);
			parser.program();
		} catch(Exception e) {
			System.err.println("exception: "+e);
		}
	}
}

File: lib/antlr-2.7.5/examples/cpp/inherit.tinyc/Main.cpp

#include <iostream>

#include "TinyCLexer.hpp"
#include "MyCParser.hpp"

int main()
{
	ANTLR_USING_NAMESPACE(std)
	ANTLR_USING_NAMESPACE(antlr)

	try {
		TinyCLexer lexer(cin);
		MyCParser parser(lexer);
		parser.program();
	}
	catch( ANTLRException& e )
	{
		cerr << "exception: " << e.getMessage() << endl;
		return -1;
	}
	catch( exception& e )
	{
		cerr << "exception: " << e.what() << endl;
		return -1;
	}
	return 0;
}

